frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/restricted-zone-popup/restricted-zone-popup.component.html

```diff
@@ -0,0 +1,277 @@
+<ng-container *ngIf="restrictedZone$ | async as restrictedZone">
+    <h5 class="popover-header">
+        {{ restrictedZone.name }}
+        <button
+            (click)="closePopup()"
+            type="button"
+            class="btn-close float-end"
+        ></button>
+    </h5>
+    <div
+        class="popover-body d-flex flex-column"
+        style="width: 500px; height: 400px"
+    >
+        <nav #nav="ngbNav" ngbNav [(activeId)]="activeNavId" class="nav-tabs">
+            <ng-container
+                ngbNavItem="settings"
+                [disabled]="(currentRole$ | async) !== 'trainer'"
```

Comment:
The popup is currently not available to non-trainers at all, so there is no need for this additional limitation